src/js/utils/Select.jsx

```diff
@@ -433,6 +435,7 @@ Select.defaultProps = {
   onEnterPress: null,
   optionRenderer: null,
   valueRenderer: null,
+  showSelectedOptionColor: null,
```

Comment:
If you had made prop type as `bool`, let's stick to it also for the default value.
```suggestion
  showSelectedOptionColor: false,
```